src/routes/api/accommodation.js

```diff
@@ -8,14 +8,16 @@ const { validate, Authenticate } = middlewares;
 const { verifyToken, verifyTravelAdmin } = Authenticate;
 
 const {
-  createAccommodation, getAccommodation, getAccommodationById,
-  updateAccommodation, deleteAccommodation
+  createAccommodation, getAccommodation, getAccommodationById, updateAccommodation,
+  deleteAccommodation, likeAccommodation, dislikeAccommodation
 } = AccommodationController;
 
 router.post('/accommodations', verifyToken, verifyTravelAdmin, validate('createAccommodation'), createAccommodation);
 router.get('/accommodations', verifyToken, getAccommodation);
 router.get('/accommodations/:accommodationId', verifyToken, validate('checkAccommodationId'), getAccommodationById);
 router.put('/accommodations/:accommodationId', verifyToken, verifyTravelAdmin, validate('updateAccommodation'), updateAccommodation);
 router.delete('/accommodations/:accommodationId', verifyToken, verifyTravelAdmin, validate('checkAccommodationId'), deleteAccommodation);
+router.patch('/accommodations/:accommodationId/like', verifyToken, validate('checkAccommodationId'), likeAccommodation);
```

Comment:
This should be `PUT` or `POST` preferably.

Reply:
Changed